cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -216,7 +217,8 @@ func textFormatPrintAddedOutput(obj *api.AddedOutput) {
 }
 
 func textFormatPrintMetric(obj *api.Metric) {
-	fmt.Printf("%s: %s | Expire : %d\n", obj.Peer.Pretty(), obj.Value, obj.Expire)
+	date := time.Unix(0, obj.Expire).UTC().Format(time.RFC3339)
+	fmt.Printf("%s: %s | Expire: %s\n", obj.Peer, obj.Value, date)
```

Comment:
@hsanjuan Why not just decode the `peer.ID` here instead of doing it everywhere else in PR?

Reply:
Not really or I misunderstood.

The point I was trying to make is that now have to do a decode operation every time you want to store the peer, which is a lossy operation in the type system sense, instead of making peer.ID.Pretty() work properly or making a peer.ID.JSON() function which would be called here and there be the changes all over the metrics code. Does that make sense? To summarise, can we fix the output of the value so we aren't having to do it on the input

Reply:
@lanzafame but I'm fixing the output of the `/monitor/metrics` endpoint, it's not just the user output here (which did not work anyway because I don't think json knows how to deserealize byte slices properly)

Reply:
i.e. I don't think this is acceptable (and doesn't work):

```
[10:49:37] ~ $ curl -s localhost:9094/monitor/metrics/ping | jq
[
  {
    "Name": "ping",
    "Peer": "\u0012 \u000bȇ�\u001a�4�����J���\r4�_U�E5�)xݭD�\u001c",
    "Value": "",
    "Expire": 1540457399324099800,
    "Valid": true
  }
]
```

Reply:
Right yep. I still prefer the struct bring an peer id because we pass it around so much. We could just define a json marshaller for the metric struct so it handles it properly

Reply:
I'll go the `MetricSerial` way then, for consistency with the rest. Since we convert object into serializable-Go objects, which can then be encoded not only as json, but also with more efficient codecs (like the go-codec thing we use for rpcs etc). I don't know if we should change that, but if we do I'd like it changed everywhere.

Reply:
Actually I'll check json marshaler/unmarshaler too, first. Maybe worth making an exception because this type is serializable, it's just json not working well for it.